internal/lido/contracts/csfeedistributor/rewards_test.go

```diff
@@ -113,3 +113,72 @@ func TestConvertTreeValuesToBigInt(t *testing.T) {
 		}
 	}
 }
+
+func TestBinarySearchNodeID(t *testing.T) {
```

Comment:
Cover an empty `treeData.Values` please. This is something very fuzzable (would benefit a lot from it)

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/402/commits/d30a7e77d2a690d8ae6916c2106b3b60bf402c57).
PS: I wasn't able to create a fuzzy test since `treeData` is an invalid fuzzy argument.

Reply:
then we should generate random slices, and purposely insert or remove/skip the desired value that is searched for